CMakeLists.txt

```diff
@@ -156,9 +156,9 @@ else()
         set(RTTR_DOCDIR "${DEFAULT_DOCDIR}" CACHE STRING "Directory for documentation files")
         if("${RTTR_LIBDIR}" STREQUAL "")
             # Avoid using absolute path accidentally
-            set(DEFAULT_DRIVERDIR "driver")
+            set(DEFAULT_DRIVERDIR "s25rttr/driver")
```

Comment:
this will create weird directories on windows.

Reply:
No it actually makes perfect sense. Windows is handled above. So this is linux only.

Reply:
ah you're right. then its good